File: 0001-two-sum/0001-two-sum.java
class Solution {
    public int[] twoSum(int[] nums, int target) {
        
        HashMap<Integer, Integer> mp = new HashMap<Integer,Integer>();

        for(int i = 0; i < nums.length; i++){
            int tmp = target - nums[i];
            if(mp.containsKey(tmp)){
                return new int[]{mp.get(tmp), i};
            } 
            mp.put(nums[i], i);
        }
        return new int[]{};
    }
}

File: 0001-two-sum/0001-two-sum.go
func twoSum(nums []int, target int) []int {
    
    var res []int

    for i, val := range nums{
        for j := i + 1; j < len(nums); j++{
            if val + nums[j] == target {
                res = append(res, i, j)
                return res
            }
        }
    }
    return res
}